lib/elixir/lib/kernel.ex

```diff
@@ -305,6 +305,10 @@ defmodule Kernel do
   prefer `module.function(arg_1, arg_2, ..., arg_n)` as it is clearer than
   `apply(module, :function, [arg_1, arg_2, ..., arg_n])`.
 
+  The function to be invoked must be a public function in `module`,
+  `apply/3` cannot be used to invoke private functions, even to call
+  private functions in the same `module` as the call to `apply/3`.
```

Comment:
Maybe  "even to call private functions in the same `module`"
is more understandable without the " as the call to `apply/3`" part added at the end?